app/_src/gateway/kong-manager/auth/sessions.md

```diff
@@ -133,3 +174,35 @@ admin_gui_session_conf = {
     "cookie_secure":false
 }
 ```
+
+{% if_version gte:3.4.x %}
+If you want to invalidate all sessions when the password is changed, the following configuration could be used for Basic Auth:
+
+```
+enforce_rbac = on
+admin_gui_auth = basic-auth
+admin_gui_session_conf = {
+    "cookie_name":"kong_cookie",
+    "secret":"change-this-secret",
+    "storage":"kong",
+    "cookie_secure":false,
+    "store_metadata": true
+}
+```
+
+If enabling the `store_metadata` and for PII reasons, the following configuration could be used instead.
```

Comment:
```suggestion
If you need to avoid storing session subjects as clear text due to PII concerns with store_metadata enabled, the following configuration can be used instead:
```